Core/HLE/sceKernel.cpp

```diff
@@ -678,23 +679,23 @@ struct DebugProfilerRegs {
 	u32 local_bus;
 };
 
-static u32 sceKernelReferThreadProfiler(u32 statusPtr) {
-	ERROR_LOG(SCEKERNEL, "FAKE sceKernelReferThreadProfiler()");
-
-	// Can we confirm that the struct above is the right struct?
-	// If so, re-enable this code.
-	//auto regs = PSPPointer<DebugProfilerRegs>::Create(statusPtr);
-	// TODO: fill the struct.
-	//if (regs.IsValid()) {
-	//	memset((DebugProfilerRegs *)regs, 0, sizeof(DebugProfilerRegs));
-	//	regs.NotifyWrite("ThreadProfiler");
-	//}
+static u32 sceKernelReferThreadProfiler() {
+	// This seems to simply has no parameter:
+	// https://pspdev.github.io/pspsdk/group__ThreadMan.html#ga8fd30da51b9dc0507ac4dae04a7e4a17 , 
+	// And in testing it just returns null in 55 usec (which is surprisingly long).
+
+	// So, we log only if debug logging is enabled, and sleep for a bit.
+	DEBUG_LOG(SCEKERNEL, "0=sceKernelReferThreadProfiler()");
+
+	// The delay has been measured, 53-56 us.
+	hleEatMicro(55);
```

Comment:
Probably not important, but I usually use hleEatCycles for things that get faster when you set the PSP clock speed faster.  This is probably (but not certainly) one of those cases.

-[Unknown]